src/main/resources/assets/autumnity/lang/zh_cn.json

```
@@ -77,11 +117,22 @@
   "biome.autumnity.maple_forest": "枫树林",
   "biome.autumnity.maple_forest_hills": "山地枫林",
   "biome.autumnity.pumpkin_fields": "南瓜原野",
+  "biome.autumnity.boreal_maple_forest": "北境枫林",
+  "biome.autumnity.boreal_maple_forest_hills": "北境山地枫林",
+
+  "subtitles.entity.snail.eat": "蜗牛：进食",
+  "subtitles.entity.snail.hurt": "蜗牛：受伤",
+  "subtitles.entity.snail.death": "蜗牛：死亡",
+  "subtitles.entity.snail.step": "蜗牛：爬行",
   
-  "subtitles.entity.snail.eat": "蜗牛进食",
-  "subtitles.entity.snail.hurt": "蜗牛受伤",
-  
+  "subtitles.entity.turkey.ambient": "火鸡：咯咯叫",
+  "subtitles.entity.turkey.aggro": "火鸡：愤怒地咯咯叫",
+  "subtitles.entity.turkey.hurt": "火鸡：受伤",
+  "subtitles.entity.turkey.death": "火鸡：死亡",
+  "subtitles.entity.turkey.egg": "火鸡：下蛋的噗噗声",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
  "subtitles.entity.turkey.egg": "火鸡：噗噗下蛋",
```

按照规范，不应出现动作声词化的现象